projects/finder-frontend/Makefile

```diff
@@ -1,2 +1,2 @@
-finder-frontend: bundle-finder-frontend content-store static search-api router email-alert-api govuk-account-manager-prototype
+finder-frontend: bundle-finder-frontend account-api content-store static search-api router email-alert-api govuk-account-manager-prototype
```

Comment:
Do we still need `govuk-account-manager-prototype` here or can we remove it since it's already in the `account-api`'s Makefile?

Reply:
There's a test that the dependencies in the Makefile match the dependencies in the docker-compose file.

Reply:
Can it be removed from both and still work as an implicit dependency that comes from `account-api`? If so, is there any value in still keeping it explicit?

Reply:
Currently finder-frontend still does directly call the account manager in places, so I think there's value in keeping it as an explicit dependency until those bits are removed.

Reply:
I see, thanks for clarifying.